docs/conf.py

```diff
@@ -92,7 +92,7 @@
 .. |minimum_numpy_version| replace:: {0.__minimum_numpy_version__}
 
 .. Astropy
-.. _Astropy: http://astropy.org
+.. _Astropy: https://astropy.org
```

Comment:
change it back for now, it should solve the issue (until we have a new sphinx-astropy out). I'll try to hack a quick hack session to tag both that one and the pytest plugins

Reply:
it's indeed defined in sphinx-astropy and here, but there is no complaint until they are they same. It has been changes in the dev version over there, but the released one still has `http`. So we'll get the error once upgrading to use the newer (not yet released) version. So if you want, we can remove the reference, or just keep the links in sync with each other. 

Consistency-wise, for this PR, I think keeping the sync makes more sense as these changes are not strictly necessary.